pandas/tests/indexes/test_category.py

```diff
@@ -540,6 +540,13 @@ def test_reindex_dtype(self):
         tm.assert_numpy_array_equal(indexer,
                                     np.array([0, 3, 2], dtype=np.intp))
 
+        with pytest.raises(ValueError, match='non-unique indexer'):
```

Comment:
can you make a new test & add the issue number as a comment

Reply:
Done